include/nbl/builtin/hlsl/scan/default_scheduler.hlsl

```diff
@@ -58,17 +42,17 @@ namespace scheduler
 	 * |> finishedFlagOffset - an index in the scratch buffer where each virtual workgroup indicates that ALL its invocations have finished their work. This helps 
 	 *							synchronizing between workgroups with while-loop spinning.
 	 */
-	void computeParameters(in uint elementCount, out Parameters_t _scanParams, out DefaultSchedulerParameters_t _schedulerParams)
+	void computeParameters(NBL_CONST_REF_ARG(uint32_t) elementCount, out Parameters_t _scanParams, out DefaultSchedulerParameters_t _schedulerParams)
```

Comment:
there's a struct with methods in CScanner that does the same thing, deduplicate it